doc/whats_new/upcoming_changes/sklearn.metrics/27412.fix.rst

```diff
@@ -1,3 +1,3 @@
-- :func:`metrics.roc_auc_score` will now correctly return 0.0 and
+- :func:`metrics.roc_auc_score` will now correctly return np.nan and
   warn user if only one class is present in the labels.
-  By :user:`Gleb Levitski <glevv>`
+  By :user:`Gleb Levitski <glevv>` and :user:`Janez Demšar <janezd>`
```

Comment:
@lesteve this PR number is not gonna show up here. What do we do?

Reply:
I think we can duplicate the changelog entry into two files with different PR numbers in their name but the same contents.

Then both entries will be merged by towncrier when we aggregate the changelog of a given release.

Reply:
Yeah that's the towncrier's way of doing it. I originally thought it was a hidden towncrier feature but it is mentioned in the [tutorial](https://towncrier.readthedocs.io/en/stable/tutorial.html#creating-news-fragments)

```
$ towncrier create --content 'Can also be ``rst`` as well!' 3456.doc.rst
# You can associate multiple issue numbers with a news fragment by giving them the same contents.
$ towncrier create --content 'Can also be ``rst`` as well!' 7890.doc.rst
```

I originally became aware of this by looking at the towncrier source code. Note that it only works if both fragments have the same type (`fix` in this case) IIRC.

There is https://github.com/twisted/towncrier/issues/599 to try to make it a bit more convenient.

Reply:
@glemaitre you've enabled automerge before resolving this though.